Remove the first empty line to enable PYTHON_SCRIPT_SHEBANG_FIX_FUNC to fix the
shebang.

Not for upstream.

--- a/base/imageprocessing.py	2020-06-04 17:10:53.315885774 +0200
+++ b/base/imageprocessing.py	2020-06-04 17:10:57.856265911 +0200
@@ -1,4 +1,3 @@
-
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
